### ISSUE TITLE
General Question about AtomTyping

### ISSUE THREAD

**Issue by GregorySchwing** [gnina/libmolgrid#116]
I am working through the "grid a single molecule" tutorial, with a different molecule from a smile string.

https://github.com/GregorySchwing/libmolgrid/blob/development/tutorials/gridmolecule_smiles.ipynb

The molecule is 

![image](https://github.com/gnina/libmolgrid/assets/39970712/445544ce-a847-4954-b3e2-47570c37179e)

being gridded into multiple different channels. 

![Figure 2(1)](https://github.com/gnina/libmolgrid/assets/39970712/ced0c3ef-2648-4d1a-9d17-c4aeb68d8970)

I am confused as to what each channel in the tensor represents.

gmaker = molgrid.GridMaker()
dims = gmaker.grid_dimensions(molgrid.defaultGninaLigandTyper.num_types())
gridtensor = torch.zeros(dims,dtype=torch.float32)
#forward( (GridMaker)arg1, (float3)arg2, (CoordinateSet)arg3, (Grid4fCUDA)arg4) -> None :
#Generate grid tensor from atomic data. Grid (GPU) must be properly sized.
gmaker.forward(c.center(),c,gridtensor)
print(gridtensor.shape)
torch.Size([14, 48, 48, 48])

I think the axis, of length 14, is the different atom types.  Is that correct?

However, the molecule only consists of C, H, O, which is 3 different atom types?

Any insight into what's going on would be appreciated.

**Comment by dkoes** [comment]
Yes, the first axis are the atom types.  Which channel is which atom type is predetermined.  Think of an image - if there was no red in it would you expect it to only have two channels instead of RGB?
Also, the default atom types have four different types of carbon and two types of oxygen, so you might have as many as 6 channels with non zero values.

**Comment by GregorySchwing** [comment]
That makes sense.  I appreciate your response.
